spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +52,24 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+def test_copy_paste_files_or_paths(qtbot, tmpdir):
+    tmpdir.join('script.py').ensure()
+    window = DirView()
+    window.show()
+    qtbot.addWidget(window)
+    fnames = [osp.join(tmpdir, 'script.py')]
+    window.copy_path(fnames=fnames, method='absolute')
+    cb = QApplication.clipboard()
+    #  test copy absolute path
+    asb_path = cb.text(mode=cb.Clipboard)
+    cb.clear(mode=cb.Clipboard)
```

Comment:
The function your testing should successfully clear the clipboard by itself, so this shouldn't be necessary and in fact is part of what you're testing (that the function works correctly with both a clean and dirty clipboard).